packages/commonwealth/client/styles/components/component_kit/cw_component_showcase.scss

```diff
@@ -273,4 +273,19 @@ $width: 360px;
       margin-bottom: 15px;
     }
   }
+
+  .auth-buttons {
```

Comment:
it does not look good 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/fa2aa853-05c3-4ee5-9106-93b63fda9fa7)

Reply:
added to new components page, not breaking now